### DISCUSSION TITLE
MongoDB Compability

### DISCUSSION THREAD

**Discussion by dinanrm** [Biarity/Sieve#203] [Q&A]
Hi,

Is it possible to use Sieve alongside MongoDB?
I'm afraid this is only compatible with SQL & EF.

Thank you

**Comment by ameckl** [comment]
@dinanrm Yes, it is possible to use Sieve on any `IQueryable`.

```c#
IMongoQueryable<Blog> queryable = MyMongoCollection.AsQueryable();

IQueryable<Blog> updatedQueryable = _sieveProcessor.Apply(sieveModel, queryable);

```
